bot/events/voiceChannelJoin.js

```diff
@@ -0,0 +1,19 @@
+const {EventListener} = require('yuuko');
+module.exports = new EventListener('voiceChannelJoin', (user, channel) => {
```

Comment:
Since [`User`](https://abal.moe/Eris/docs/User) and [`Member`](https://abal.moe/Eris/docs/Member) are different classes in Eris, it's best to name these variables after the thing they contain to avoid confusion. I'd change this to `member`.